test/controllers/guesses_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class GuessesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
You'll need to delete this file as you're testing this controller elsewhere. There are a lot of unused files that you need to delete actually.